sympy/__init__.py

```diff
@@ -85,7 +85,8 @@ def __sympy_debug():
 from .printing import python, print_python, srepr, sstr, sstrrepr
 from .interactive import init_session, init_printing
 
-evalf._create_evalf_table()
+from .core.evalf import _create_evalf_table
+_create_evalf_table()
 
```

Comment:
You need to `del` it here so it doesn't get imported.

Reply:
I think it is not necessary because `_create_evalf_table()` is started with underscore.